site/en/codelabs/openthread-border-router-nat64/index.lab.md

```diff
@@ -65,54 +65,38 @@ The network topology for this codelab:
 Duration: 07:00
 
 
-Follow the  [Setup OTBR step of the Thread Border Router - Bidirectional IPv6 Connectivity and DNS-Based Service Discovery codelab](https://openthread.io/codelabs/openthread-border-router#1) to build the OpenThread border router, with the following change:
+The quickest way to set up an OTBR is by using Docker following the  [OTBR with Docker Guide](https://openthread.io/guides/border-router/docker).
 
-In **Build and install OTBR**, you need to tell the script to enable the NAT64 translator in OpenThread by setting the environment variable `NAT64` to `1` and `NAT64_SERVICE` to `openthread`. Run the following command before the step:
+After OTBR setup is complete, use `ot-ctl` to validate that the NAT64 service is enabled on the border router:
```

Comment:
Is this still correct to say "use ot-ctl" here based on all the code removals?

Reply:
`ot-cli` is necessary to access the OpenThread CLI on OTBR. I added this link: https://openthread.io/guides/border-router/docker/run#run_ot-ctl

Does that help clarify?